stages/org.osbuild.testmac

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python3
+"""
+Enable osbuild mac test service
+
+Creates a test service with the osbuild_mac_t label and tests this services permissions.
+"""
+
+import subprocess
+import sys
+import os
+import re
+
+def main():
+
+    pid = '/proc/' + str(os.getpid()) + '/status'
+    temp = subprocess.Popen(['grep', 'CapBnd', pid], stdout = subprocess.PIPE)
+    # get the output as a string
+    output = str(temp.communicate())
+    output = re.split(',|:|\\\\n|\\\\t|\'|\\(b\'|None\\)' , output)
```

Comment:
I think I know what this is trying to do, but my regex foo is not really able to verify it. Could you explain what it does in a comment (and why all these delimiters are included in the regex, is the format defined somewhere)?

Reply:
Sure, ill add an inline comment!
FYI the escape characters look this way due to the format that subprocess.communicate() returns, its a lot of special characters that need to be escaped and the end result looks like my finger got stuck on the \ key :laughing:

Reply:
I have added an inline comment, let me know if its not clear and I can reword :smile: